crates/fire_dynamics_tools/src/chapter_9/equation_9_4.rs

```diff
@@ -0,0 +1,27 @@
+pub fn effective_diameter(a_f: f64) -> f64 {
+    (4.0 * a_f / std::f64::consts::PI).powf(0.5)
+}
+
+pub fn effective_diameter_equation(d: String, a_f: String) -> String {
+    format!("{} = (\\frac{{4 \\cdot {}}}{{\\pi}})^{{1/2}}", d, a_f)
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_effective_diameter() {
+        let a_f = 4.0;
+        let expected = 2.256758334;
```

Comment:
Same comment using the examples provided to show that we are correct:
```suggestion
        let a_f = 3.205;
        let expected = 2.02;
```